[OLD] New Readme design (#58)
colinhartigan/valorant-rpc: README.md

```
 _   _____   __   ____  ___  ___   _  ________                
| | / / _ | / /  / __ \/ _ \/ _ | / |/ /_  __/__________  ____
| |/ / __ |/ /__/ /_/ / , _/ __ |/    / / / /___/ __/ _ \/ __/
|___/_/ |_/____/\____/_/|_/_/ |_/_/|_/ /_/     /_/ / .__/\__/ 
                                                  /_/         
```
[![Discord][discord-shield]][discord-url]
[![License][license-shield]][license-url]
[![Stars][stars-shield]][stars-url]
[![Forks][forks-shield]][forks-url]
[![Releases][releases-shield]][releases-url]
[![Language][language-shield]][language-url]

<!-- TABLE OF CONTENTS -->
<details open="open">
  <summary>Table of Contents</summary>
  <ol>  
    <li><a href="demo">Demo</a></li>
    <li><a href="installation">Installation</a></li>
    <li><a href="#usage">Usage</a></li>
    <li><a href="#license">License</a></li>
    <li><a href="#support">Support</a></li>
    <li><a href="#disclaimer">Disclaimer</a></li>
  </ol>

 
 
<!-- DEMO --> 
## Demo
<a>
    <img src="assets/Demo1.png" alt="Demo" width="205" height="112">
    <img src="assets/Demo2.png" alt="Demo" width="205" height="112">
</a>

 
 
<!-- INSTALLATION --> 
## Installation

> - Download the latest [release](https://github.com/colinhartigan/valorant-rpc/releases/) and run it.

Reviewer: Change the link to: https://github.com/colinhartigan/valorant-rpc/releases/latest/download/valorant-rpc.exe

This link automatically downloads the latest .exe

```
 _   _____   __   ____  ___  ___   _  ________                
| | / / _ | / /  / __ \/ _ \/ _ | / |/ /_  __/__________  ____
| |/ / __ |/ /__/ /_/ / , _/ __ |/    / / / /___/ __/ _ \/ __/
|___/_/ |_/____/\____/_/|_/_/ |_/_/|_/ /_/     /_/ / .__/\__/ 
                                                  /_/         
```
[![Discord][discord-shield]][discord-url]
[![License][license-shield]][license-url]
[![Stars][stars-shield]][stars-url]
[![Forks][forks-shield]][forks-url]
[![Releases][releases-shield]][releases-url]
[![Language][language-shield]][language-url]

> <!-- TABLE OF CONTENTS -->

Reviewer 2: ```suggestion
```

<details open="open">
  <summary>Table of Contents</summary>
  <ol>  
    <li><a href="explanation">Explanation</a></li>

>     <li><a href="#installation">Installation</a></li>

Reviewer: ```suggestion
    <li><a href="#About">About</a></li>

```

> <details open="open">
>   <summary>Table of Contents</summary>
>   <ol>  
>     <li><a href="explanation">Explanation</a></li>
>     <li><a href="#installation">Installation</a></li>
>     <li><a href="#usage">Usage</a></li>
>     <li><a href="#license">License</a></li>
>     <li><a href="#support">Support</a></li>
>     <li><a href="#disclaimer">Disclaimer</a></li>
>   </ol>

Reviewer 2: Can I ask why whole readme is in Table of Contents too? If you will close ToC it will not show readme.
```suggestion
<details open="open">
  <summary>Table of Contents</summary>
  <ol>  
    <li><a href="explanation">Explanation</a></li>
    <li><a href="#installation">Installation</a></li>
    <li><a href="#usage">Usage</a></li>
    <li><a href="#license">License</a></li>
    <li><a href="#support">Support</a></li>
    <li><a href="#disclaimer">Disclaimer</a></li>
  </ol>
</details>
``` 
Fixed with this code.

Reviewer 3: I forgot to add it

> <!-- DEMO --> 

Reviewer 2: ```suggestion
```

## Explanation?
RPC (Rich Presence) is what allows discord to show others what you are currently doing within supported applications!

 <a>
    <img src="assets/Demo1.png" alt="Demo" width="205" height="112">
    <img src="assets/Demo2.png" alt="Demo" width="205" height="112">

> </a>

Reviewer 2: What's the point of using `<a>` tag here? Oh also, will be nice if you would upload these images.

Reviewer 3: Image links are in the first comment of the thread

Reviewer: > Image links are in the first comment of the thread

Add them to the Assets folder so it goes along with this PR

Reviewer 3: > What's the point of using `<a>` tag here? Oh also, will be nice if you would upload these images.

What is a better way to do this whilst still having it link elsewhere?

> <!-- INSTALLATION --> 

Reviewer 2: ```suggestion
```

## Installation
- Download the latest [release](https://github.com/colinhartigan/valorant-rpc/releases/latest/download/valorant-rpc.exe) and run it.

 
 

> <!-- USAGE -->

Reviewer 2: ```suggestion
```

## Usage
- Run the program instead of launching VALORANT
     - If VALORANT is not running, the program will launch it for you
- If VALORANT is already running, launch the program and the presence will start

 

> <!-- LICENSE -->

Reviewer 2: ```suggestion
```

## License

> Click [here](license-url) for the license.

Reviewer 2: that's kinda unnecessary to have in readme.

> <!-- SUPPORT -->

Reviewer 2: ```suggestion
```

## Support
Either make an issue or:
[![Discord][discord-shield]][discord-url]
 
 
 

> <!-- DISCLAIMER -->

Reviewer 2: ```suggestion
```

## Disclaimer 
This project is not affiliated with Riot Games or any of its employees and therefore does not reflect the views of said parties. This is purely a fan-made project to enhance VALORANT's skin inventory management.

Riot Games does not endorse or sponsor this project. Riot Games, and all associated properties are trademarks or registered trademarks of Riot Games, Inc.
 
 
 
<!-- MARKDOWN LINKS & IMAGES -->

> [discord-shield]: https://img.shields.io/discord/860288779558715402?color=7289da&label=Support&logo=discord&logoColor=7289da&style=for-the-badge

Reviewer 2: Widget is disabled so 🤷

Reviewer 4: i turned them on

 
 
<!-- DEMO --> 

> ## Explanation?
> RPC (Rich Presence) is what allows discord to show others what you are currently doing within supported applications!

Reviewer: I was thinking more like:
```suggestion
## About
Valorant RPC (Rich Presence) allows you to show in-game details such as the current score in your Discord Profile! Additional features include showing current map, agent, idle status, etc.
```
